src/app/common/stores/applications/SsvWeb/Application.store.ts

```diff
@@ -74,6 +73,11 @@ class ApplicationStore extends BaseStore implements Application {
     } else {
       this.switchDarkMode(false);
     }
+
+    console.info('Checking "locationRestrictionDisabled" in localStorage to adjust "locationRestrictionEnabled" flag');
```

Comment:
same here

Reply:
Here console also by purpose, to remind developer on stage env what's the flag name without looking into the code.
At some point it should become part of feature flags